### PR TITLE
Updated ouroboros-network dependency

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

**Comment by coot** [comment]
@erikd sorry I am polluting `stack.yaml` file here - as this is the current state of affairs.  It will be easier to move things in one go to `cardano-prelude`'s `stack.yaml` file.

File: stack.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ extra-deps:
   - esqueleto-3.2.2
   - generic-monoid-0.1.0.0
   - libsystemd-journal-1.4.4
+  - network-3.1.0.1
   - persistent-2.10.4
   - persistent-postgresql-2.10.1
   - persistent-template-2.7.3
```
Comment (erikd) [inline stack.yaml:32]: I think most of these should be in `snapshot.yaml` of the `cardano-prelude` repo.

File: .stack-to-nix.cache [hunk 2]
```diff
@@ -29,3 +29,28 @@ https://github.com/input-output-hk/ouroboros-network 408f3a3fc8be601075ee2bacdb7
 https://github.com/input-output-hk/ouroboros-network 408f3a3fc8be601075ee2bacdb7b7b9ebac9d2c2 typed-protocols 0hc8p7h60wq4xp43pl7rqn94lra7gy51z3h6jjqvy6lj4bcfrp65 typed-protocols typed-protocols.nix
```
Comment (erikd) [inline .stack-to-nix.cache]: I really do not think this file should even be in the repo.

**Review by erikd — DISMISSED** [review]

File: stack.yaml (hunk 1 above)
Comment (dcoutts) [reply stack.yaml:32]: Yes, they should move down there and we can bubble it up. This is ok for tagging this release though.

File: .stack-to-nix.cache (hunk 2 above)
Comment (dcoutts) [reply .stack-to-nix.cache]: I agree. I think it goes away with the move to the updated haskell nix infrastructure right?

**Review by dcoutts — APPROVED** [review]
LGTM. Lib dep versions are consistent with the node.

**Pull request merged**